tests/python/pants_test/build_graph/test_build_file_parser.py

```diff
@@ -104,10 +109,11 @@ def test_invalid_unicode_in_build_file(self):
       """
     )))
     build_file = self.create_buildfile('BUILD')
-    self.assert_parser_error(build_file, 'invalid syntax')
+    self.assert_parser_error(build_file, 'invalid character' if PY3 else 'invalid syntax')
 
   def test_unicode_string_in_build_file(self):
     """Demonstrates that a string containing unicode should work in a BUILD file."""
+    # TODO(python3port): remove ensure_binary once safe_open() uses backport. This test fails on Py3.
```

Comment:
Less important here, but in general `TODO`s should be targeted at tickets rather than people or topics.

Reply:
John and I discussed use the approach to add # TODO(python3port) and then track here #6071

I didn't add these to issue because I'll be fixing it in next few days as part of migration, whereas in general this is meant to be things that are issues discovered during migration but not immediately necessary to fix.

Reply:
Then the TODO could reference #6071 presumably?